pkg/controller/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -0,0 +1,615 @@
+package toolchainstatus
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/codeready-toolchain/host-operator/pkg/configuration"
+	"github.com/codeready-toolchain/host-operator/pkg/controller/registrationservice"
+	. "github.com/codeready-toolchain/host-operator/test"
+
+	"github.com/codeready-toolchain/api/pkg/apis"
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+	"github.com/codeready-toolchain/toolchain-common/pkg/status"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	kubefed_common "sigs.k8s.io/kubefed/pkg/apis/core/common"
+	kubefed_v1beta1 "sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
+)
+
+var requeueResult = reconcile.Result{RequeueAfter: defaultRequeueTime}
+
+const defaultHostOperatorName = "host-operator"
+
+const defaultToolchainStatusName = configuration.DefaultToolchainStatusName
+
+type fakeHTTPClient struct {
+	response http.Response
+	err      error
+}
+
+func (f *fakeHTTPClient) Get(url string) (*http.Response, error) {
+	return &f.response, f.err
+}
+
+const respBodyGood = "{\"alive\":true,\"environment\":\"dev\",\"revision\":\"64af1be5c6011fae5497a7c35e2a986d633b3421\",\"buildTime\":\"0\",\"startTime\":\"2020-07-06T13:18:30Z\"}"
+const respBodyInvalid = "{\"not found\"}"
```

Comment:
Just FYI. You can use `` instead:
```suggestion
const respBodyInvalid = `{"not found"}`
```

Reply:
I should have thought of that lol. I've updated it, thanks